setup.py

```diff
@@ -6,12 +6,17 @@
     name='simple-httpfs',
     author='Peter Kerpedjiev',
     author_email='pkerpedjiev@gmail.com',
-    packages=[ 'simple_httpfs' ],
-    scripts=['simple_httpfs/simple-httpfs.py'],
+    packages=['simple_httpfs'],
+    entry_points={
+      'console_scripts': [
+          'httpfs = simple_httpfs.__main__:main'
```

Comment:
I already merged this PR, but does this mean that this will be called from the command line using `httpfs ...`?

If so, the README needs to be updated.

Reply:
Also I think the command line name should be more specific than just `httpfs`, like maybe `simple-httpfs` ;-)

Reply:
Oh yeah, I thought it would be nice if it were more succinct, but changing it back to `simple-httpfs` is fine with me.